src/tsclient/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "impress-openapi-client-ts",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
so many versions... we might forget to bump some in future releases. May I suggest to add checks in the CI to make sure they are all in sync : see https://github.com/openfun/richie/blob/master/.circleci/config.yml#L515